app/src/org/commcare/views/widgets/BarcodeWidget.java

```diff
@@ -31,7 +31,7 @@ public BarcodeWidget(Context context, FormEntryPrompt prompt, PendingCalloutInte
     }
 
     @Override
-    protected void performCallout() {
+    public void performCallout() {
```

Comment:
we should avoid changing method visibility for tests as much as possible. Instead you can just trigger the UI action that results in this being called out.

Reply:
makes sense. Will do the changes.